tests/test_model_variant_name_manager.py

```diff
@@ -17,188 +17,109 @@
 import unittest
 from unittest.mock import patch
 
+from tests.common.test_utils import default_encode
+from model_analyzer.constants import DEFAULT_CONFIG_PARAMS
+
 
 class TestModelVariantNameManager(trc.TestResultCollector):
 
     def setUp(self):
-        NotImplemented
+        self._mvnm = ModelVariantNameManager()
+        self._non_default_param_combo = {'foo': 1}
 
     def tearDown(self):
         patch.stopall()
 
     def test_default(self):
         """
-        If no param is passed in, it is considered the default config
+        Check that default config is returned
         """
-        mvnm = ModelVariantNameManager()
-        name = mvnm.get_model_variant_name("modelA", {})
-        self.assertEqual(name, "modelA_config_default")
+        name = self._mvnm.get_model_variant_name("modelA", {'A': 1},
```

Comment:
call this "result" instead of "name", since it is a tuple

Reply:
Calling this a# to be consistent with the other tests